docs/static/upgrading.asciidoc

```diff
@@ -106,87 +109,38 @@ Before upgrading Logstash:
 +
 There you can find info on these topics and more:
 
-** <<java-exec-default,Java execution engine enabled by default>>
-** <<field-ref-strict,Field parser is more strict and how that affects processing>>
-** <<beats-ecs,Beats conforms to the Elastic Common Schema (ECS) and how that impacts {ls}>>
+** <<bc-ecs-compatibility,Elastic Common Schema (ECS) compatibility on by default>>
+** Secure communication with Elasticsearch
+** <<bc-field-ref-parser,Field parser is more strict>>
+
  
 If you are installing Logstash with other components in the Elastic Stack, also see the
 {stack-ref}/index.html[Elastic Stack installation and upgrade documentation].
 
-NOTE: Upgrading between non-consecutive major versions (5.x to 7.x, for example) is not
-supported. We recommend that you upgrade to 6.x, and then upgrade to 7.x.
+NOTE: Upgrading between non-consecutive major versions (6.x to 8.x, for example) is not supported. 
+We recommend that you upgrade to {prev-major-last}, and then upgrade to 8.0.
 
-[float]
+[discrete]
 [[upgrade-to-6.8-rec]]
```

Comment:
Shouldn't we change also the title section from `[[upgrade-to-6.8-rec]]` to something like `[[upgrade-to-latest-previous-major]]` ?